obs-packaging/linux-container/kata-linux-container.spec-template

```diff
@@ -116,10 +121,18 @@ InstallKernel() {
     rm -f %{buildroot}/usr/lib/modules/$KernelVer/source
 }
 
+%ifarch ppc64le
+InstallKernel $(realpath vmlinux) $(realpath vmlinux)
```

Comment:
So ppc does not have a compressed linux file?
If so, wouldn't it make more sense in the final package to just have `/usr/share/kata-containers/vmlinux-...` ? Unless qemu needs that o.f.c.

Reply:
@marcov : ppc does not have a compressed linux file. I was not sure if qemu or other components need it or not. So followed what was in the developer guide docs. (https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#install-guest-kernel-images)

Reply:
It would be great if you could check it on a ppc host.

Reply:
@marcov : Only one should do. So we are looking for something like this?
```
%files
%dir /usr/share/kata-containers
/usr/share/kata-containers/vmlinux-%{kversion}
/usr/share/kata-containers/vmlinux.container
%ifarch x86_64
/usr/share/kata-containers/vmlinuz-%{kversion}
/usr/share/kata-containers/vmlinuz.container
%endif
```

Reply:
@marcov : Updated my PR. PTAL

Reply:
Thanks. There's a lot of conditional %if blocks, maybe when adding arm64 we should try to refactor a bit.